src/components/content/entity.tsx

```diff
@@ -172,40 +165,15 @@ export function Entity({ data, contentId, contentType, license }: EntityProps) {
     if (contentType === 'Exercise' || contentType === 'ExerciseGroup')
       return null
 
-    /* TODO: Maybe merge with icon logic in topic-link-list, Might expose a component `PageTitle` */
-
-    let icon = faCircle
-    let iconTitle = ''
-
-    switch (contentType) {
-      case 'Article':
-        icon = faNewspaper
-        iconTitle = 'Artikel'
-        break
-      case 'CoursePage':
-        icon = faGraduationCap
-        iconTitle = 'Kurs'
-        break
-      case 'Video':
-        icon = faPlayCircle
-        iconTitle = 'Video'
-        break
-      case 'Applet':
-        icon = faCubes
-        iconTitle = 'Applet'
-        break
-      case 'TaxonomyTerm':
-        icon = faFolderOpen
-        iconTitle = 'Bereich'
-    }
+    const iconAndTitle = getIconAndTitleByContentType(contentType)
```

Comment:
Is there a reason that this isn't a Component `<TitleWithIcon contentType={contentType} />`?

Reply:
Hmm, I guess. Topic uses them in a different way (h4 instead of h1) other style so I thought this is more flexible since we might also just load the icon in some places.

Reply:
Fine by me :)